src/SFA.DAS.AssessorService.Application.Api.UnitTests/Validators/ExternalApi/ExternalApiValidatorsTestBase.cs

```diff
@@ -7,19 +7,18 @@
 using SFA.DAS.AssessorService.Domain.JsonData;
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using Organisation = SFA.DAS.AssessorService.Domain.Entities.Organisation;
-
-
-namespace SFA.DAS.AssessorService.Application.Api.UnitTests.Validators.Certificates
+namespace SFA.DAS.AssessorService.Application.Api.UnitTests.Validators.ExternalApi
 {
-    public class BatchCertificateRequestValidatorTestBase
+    public class ExternalApiValidatorsTestBase
```

Comment:
This is the answer to some of my other comments - there should be comments in this (I can see some) which explain why certain test data is expected to be valid and other test data not.

Reply:
Added extra comments